cmd/blocks.go

```diff
@@ -89,7 +89,7 @@ func BlockIgnore(blockID string) error {
 }
 
 func BlockFile(blockID string, index int, path string, content bool) error {
-	urlPath := "blocks/"+blockID+"/files"
+	urlPath := "blocks/" + blockID + "/files"
```

Comment:
Pity that the go formatters don't seem to enforce this.

Reply:
They do, they just hadn't been run. All these little changes are the result of `make fmt`